PublicGitArchive/pga-create/cmd/pga-create/discover.go

```diff
@@ -70,71 +70,78 @@ func reduceWatchers(stream io.Reader) map[int]int {
 		if err != nil {
```

Comment:
instead of `Atoi` you should use `strconv.ParseUint(line[:commaPos], 10, 32)` to detect any overflow